.travis.yml

```diff
@@ -13,4 +13,5 @@ install:
   - travis_retry composer install
 
 script:
-  - phpunit
\ No newline at end of file
+  - phpunit --exclude-group integration
+  - '[[ "${TRAVIS_PULL_REQUEST}" == "false" ]] && ( phpunit --group integration ) || ( echo "Testing PR - No integration tests available")'
```

Comment:
Travis does not export the secret environment variables when running PR builds. Since this could cause confusion, it's better to output a message than just do nothing or let the integration tests on the PR fail.